app/src/main/java/com/vci/vectorcamapp/imaging/presentation/ImagingScreen.kt

```diff
@@ -288,6 +288,77 @@ fun ImagingScreen(
                     )
                 }
 
+                if (state.showProcessFurtherDialog) {
+                    AlertDialog(
+                        onDismissRequest = { },
+                        title = {
+                            Row(
+                                verticalAlignment = Alignment.CenterVertically
+                            ){
+                                Icon(
+                                    painter = painterResource(id = R.drawable.ic_info),
+                                    contentDescription = "Info Icon",
+                                    tint = MaterialTheme.colors.icon,
+                                    modifier = Modifier
+                                        .size(MaterialTheme.dimensions.iconSizeLarge)
+                                )
+                                Spacer(modifier.size(MaterialTheme.dimensions.spacingMedium))
+                                Text(
+                                    text = "Specimen Selected for Further Processing",
+                                    color = MaterialTheme.colors.icon
+                                )
+                            }
+                        },
+                        text = {
+                            Column {
+                                Text(
+                                    text = "This specimen has been randomly selected and must be packed separately for additional laboratory processing. Please package this specimen for further processing now before continuing.",
```

Comment:
This is a lot of text which is saying the same thing multiple times. Maybe compress it a bit, make it more succinct.